drivers/modem/gsm_ppp.c

```diff
@@ -697,12 +692,11 @@ static void gsm_finalize_connection(struct gsm_modem *gsm)
 		return;
 	}
 
-	/* Attached, clear retry counter */
+	/* Attached */
```

Comment:
Is this comment necessary? IMO. `gsm->attached = true;` is much enough.

Reply:
Agreed